functions/backup.bash

```diff
@@ -486,6 +488,18 @@ setup_mirror_SD() {
   if [[ $1 == "remove" ]]; then
     cond_redirect systemctl disable sdrsync.service sdrawcopy.service sdrsync.timer sdrawcopy.timer
     rm -f "$serviceTargetDir"/sdr*.{service,timer}
+
+    # ATTENTION: the mountpoint may also have a different name than the default "/storage"
+    svcname="${${storageDir:1}//[\/]/\\x2d}.mount"     # remove leading '/' and replace all '/' by \\x2d as required by systemd for full pathnames 
+    if [[ -f "$serviceTargetDir"/"$svcname" ]]; then
+        # ATTENTION: may not be desired to remove on SD mirror disabling because it may still be in use for Amanda storage => ask for confirmation
+        # [is there a possibility that this routine might be run non-interactively ?]
+        if (whiptail --title "Remove $storageDir" --yes-button "Remove" --no-button "Keep" --yesno "$storageRemovalQuery" 12 116); then 
+            cond_redirect systemctl disable --now "${svcname}"
+            rm -f "$serviceTargetDir"/"$svcname"
+        fi
```

Comment:
Should probably add a way for the user to add a custom value in the menu other than having to go change the value in the file.

Reply:
I thought about it but doing it right would require lots of extra quirks and tests in the code such as to auto-edit openhabian.conf that ain't worth the effort